src/app/work-item/work-item-quick-add/work-item-quick-add.component.html

```diff
@@ -11,19 +11,24 @@
     </div>
     <div class=" workItemQuickAdd_story">
       <div class="col-md-6 col-sm-6 col-xs-12 workItemQuickAdd_storyInput">
-        <input #quickAdd type="text" id="exampleInput" class="form-control width100 pull-left"  [(ngModel)]="workItem.fields['system.title']" 
-              (keyup.enter)="save()"
+        <input #quickAddTitle type="text" id="exampleInput" class="form-control width100 pull-left"  [(ngModel)]="workItem.fields['system.title']" 
+              (keydown.enter)="save()"
```

Comment:
why change it to keydown? that means that if you press and hold enter it fires the save